client/branded/src/global-styles/nav.scss

```
@@ -13,6 +13,24 @@ $nav-tabs-link-active-border-color: var(--border-color) var(--border-color) tran
 // to have an undesirable bottom border when active.
 .nav-tabs > .nav-item > .nav-link {
     height: 100%;
+    .theme-redesign & {
+        border: none;
+        border-radius: var(--border-radius);
+        color: var(--text-muted);
+        &:not(.active):hover {
+            border-bottom: 3px solid var(--border-color-2);
+            margin-top: 0;
+            color: var(--body-color);
+        }
+    }
+}
+
+.nav-tabs > .nav-item > .nav-link.active {
+    .theme-redesign & {
+        color: var(--body-color);
+        font-weight: bold;
```

It seems that according to [mockups](https://www.figma.com/file/NIsN34NH7lPu04olBzddTw/Design-Refresh-Systemization-source-of-truth?node-id=1107%3A23176), it should be `600`.

```suggestion
        font-weight: 600;
```